src/vue/components/word-usage-examples/word-usage-examples-filters.vue

```diff
@@ -79,7 +80,7 @@ export default {
       return this.$store.state.app.homonymDataReady ? this.app.homonym : null
     },
     languageCode () {
-      return  this.homonym ? this.homonym.language : null
+      return  this.homonym ? LanguageModelFactory.getLanguageCodeFromId(this.homonym.languageID) : null
```

Comment:
And the same question (simply not to loose it :) ) - why did your remove language property?

Reply:
Currently all references to Homonym.language emit a console warning because we had decided that use of this property should be deprecated.  I could have removed the deprecation warning, but the whole point was to try to see where we were using it in the code and correct it.  Ultimately we still a lot of cleanup to do around how we handle the language code throughout -- this was postponed out of this release but I still want to do it. E.g. see  alpheios-project/data-models#65